ops/pebble.py

```diff
@@ -2769,6 +2798,10 @@ def exec(
         combine_stderr: bool = False,
     ) -> ExecProcess[bytes]: ...
 
+    # FIXME: would be kinda nice to log the command
+    # but what if the command contains some password?
+    # maybe log the first element of command?
```

Comment:
It definitely feels very risky to be including the full command -- I would say it's much better to let the charm instrument that if they really need it.

The first element does *seem* like it should always be safe (and would provide most of the use).

Reply:
logging first element now.